tools/src/main/java/org/apache/rocketmq/tools/command/acl/ClusterAclConfigVersionListSubCommand.java

```diff
@@ -0,0 +1,131 @@
+    }
+
+    @Override public Options buildCommandlineOptions(Options options) {
+        OptionGroup optionGroup = new OptionGroup();
+
+        Option opt = new Option("b", "brokerAddr", true, "query acl config version for which broker");
+        optionGroup.addOption(opt);
+
+        opt = new Option("c", "clusterName", true, "query acl config version for specified cluster");
+        optionGroup.addOption(opt);
+
+        optionGroup.setRequired(true);
+        options.addOptionGroup(optionGroup);
+        
+        return options;
+    }
+
+    @Override public void execute(CommandLine commandLine, Options options,
+        RPCHook rpcHook) throws SubCommandException {
+
+        DefaultMQAdminExt defaultMQAdminExt = new DefaultMQAdminExt(rpcHook);
+        defaultMQAdminExt.setInstanceName(Long.toString(System.currentTimeMillis()));
+
+        try {
+
+            if (commandLine.hasOption('b')) {
+                String addr = commandLine.getOptionValue('b').trim();
+                defaultMQAdminExt.start();
+                printClusterBaseInfo(defaultMQAdminExt, addr);
+
+                System.out.printf("get broker's plain access config version success.%n", addr);
+                return;
+
+            } else if (commandLine.hasOption('c')) {
+                String clusterName = commandLine.getOptionValue('c').trim();
+
+                defaultMQAdminExt.start();
+
+                Set<String> masterSet =
+                    CommandUtil.fetchMasterAddrByClusterName(defaultMQAdminExt, clusterName);
+                System.out.printf("%-16s  %-22s  %-22s  %-20s  %-22s%n",
+                    "#Cluster Name",
+                    "#Broker Name",
+                    "#Broker Addr",
+                    "#AclConfigVersionNum",
+                    "#AclLastUpdateTime"
+                );
+                for (String addr : masterSet) {
+                    printClusterBaseInfo(defaultMQAdminExt, addr);
+                }
+                System.out.printf("get cluster's plain access config version success.%n");
+
+                return;
+            }
+
+            ServerUtil.printCommandLineHelp("mqadmin " + this.commandName(), options);
+        } catch (Exception e) {
+            throw new SubCommandException(this.getClass().getSimpleName() + " command failed", e);
+        } finally {
+            defaultMQAdminExt.shutdown();
+        }
+    }
+
+    private void printClusterBaseInfo(
+        final DefaultMQAdminExt defaultMQAdminExt, final String addr) throws
+        InterruptedException, MQBrokerException, RemotingException, MQClientException {
+
+
+        ClusterAclVersionInfo clusterAclVersionInfo = defaultMQAdminExt.examineBrokerClusterAclVersionInfo(addr);
+        DataVersion aclDataVersion = clusterAclVersionInfo.getAclConfigDataVersion();
+        String versionNum = String.valueOf(aclDataVersion.getCounter());
+
+        DateFormat sdf = new SimpleDateFormat(UtilAll.YYYY_MM_DD_HH_MM_SS);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.